docs/user-guide/examples/advanced_examples/advanced_brokers_hierarchies.md

```diff
@@ -48,16 +48,15 @@ When running across multiple compute nodes, the relevant portion of the helics_c
 
 `broker_hierarchy_runner_A.json`
 
-```
+```json
```

Comment:
This isn't valid JSON; why do we need to add the JSON tag?

Reply:
The markdown linter wants all code blocks to have a language given. Since its an excerpt from a JSON file, JSON seems to be a better match than other choices like `shell`.

For reference, the languages supported by GitHub markdown are: https://www.rubycoloredglasses.com/2013/04/languages-supported-by-github-flavored-markdown/

Reply:
Another idea was that the `...` before and after the JSON fragment could be removed since those are never valid lines in a JSON file. Earlier on the docs say `the relevant portion of the helics_cli runner files would look like this` so readers should be able to tell that it is just a single line from a larger file.

Reply:
You can use `plaintext`:

````
```plaintext
This code will not be syntax highlighted.
```
````